public/admin/config.yml

```diff
@@ -153,3 +153,72 @@ collections:
       - { label: "Video Link", name: "videoLink", widget: "string", required: false }
       - { label: "Slides", name: "slides", widget: "file", required: false, hint: "Presentation slides." }
       - { label: "Register Link", name: "registerLink", widget: "string", required: false }
+  - name: "carousel"
+    label: "Carousel"
+    files:
+      - label: "Home Carousel"
+        name: "home-carousel"
+        file: "cms/carousel/home-carousel.md"
+        editor:
+          preview: false
+        fields:
+          - label: "Slides"
+            name: "slides"
+            widget: "list"
+            fields:
+              - label: "Title"
+                name: "title"
+                widget: "string"
+              - label: "Subheader"
+                name: "subheader"
+                widget: "string"
+                required: false
+              - label: "Description"
+                name: "description"
+                widget: "text"
+              - label: "Background image"
+                name: "backgroundImage"
+                widget: "file"
+                required: false
```

Comment:
make required instead? then no need to do the check in the component

Reply:
just background image only? i was thinking of keeping optional so can still publish without it